### PR TITLE
Bumping mqtt dependency

### LINKED ISSUE #60: MQTT listener goes 'stale'
So I'm running mqtt2prometheus and it seems to generally work, but after some time, no more messages are received from mqtt.

The prometheus exporter side is still working (as prometheus reports the target as 'up').

I've enabled debug messaging, but nothing really comes up.

What kinda extra stuff can I enable? Happy to hack away and built it myself with extra debug logging enabled or something.

My guess is probably the component that listens to the mqtt topic has gone stale/timed out/needs to reconnect, but it never does and it just sits there doing nothing.

Thoughts? Might stare at the code later, just raising this as a first step :)

### PR DESCRIPTION
**Pull request by michael-robbins**
Testing out bumping the mqtt dependency to try and fix #60

### CODE REVIEW

**Comment by michael-robbins** [comment]
Docker image builds, I'm currently running it on my servers, and it seems to at least 'still work', so that's a good thing, will run it for a few days and see if I experience any random disconnects

File: go.sum [hunk 1]
```diff
@@ -105,6 +107,8 @@ github.com/gorilla/context v1.1.1/go.mod h1:kBGZzfjB9CEq2AlWe17Uuf7NDRt0dE0s8S51
 github.com/gorilla/mux v1.6.2/go.mod h1:1lud6UwP+6orDFRuTfBEV8e9/aOM/c4fVVCaMa2zaAs=
 github.com/gorilla/mux v1.7.3/go.mod h1:1lud6UwP+6orDFRuTfBEV8e9/aOM/c4fVVCaMa2zaAs=
 github.com/gorilla/websocket v0.0.0-20170926233335-4201258b820c/go.mod h1:E7qHFY5m1UJ88s3WnNqhKjPHQ0heANvMoAMk2YaljkQ=
+github.com/gorilla/websocket v1.4.2 h1:+/TMaTYc4QFitKJxsQ7Yye35DkWvkdLcvGKqM+x0Ufc=
+github.com/gorilla/websocket v1.4.2/go.mod h1:YR8l580nyteQvAITg2hZ9XVh4b55+EU/adAjf1fMHhE=
```
Comment (michael-robbins) [inline go.sum:111]: Was brought in as a dependency from eclipse/paho.mqtt.golang, can verify this by checking their repo: https://github.com/eclipse/paho.mqtt.golang/blob/master/go.mod

**Comment by michael-robbins** [comment]
Should be good to review and test @hikhvar, working on my setup at least, I saw it reconnect overnight at 2am by itself and later in the day yesterday.

And it successfully started receiving device messages this morning when my devices started messaging again.

**Comment by hikhvar** [comment]
Thanks for the debugging and the PR. I will fix the CI later and create a Bugfix release.

**Comment by hikhvar** [comment]
Just released v0.1.6-RC8. 

I need some docs rework before releasing v0.1.6 finally.

**Pull request merged**